packages/disputer/index.js

```diff
@@ -30,7 +30,7 @@ const { getAbi, getAddress } = require("@umaprotocol/core");
  * @param {String} [disputerOverridePrice] Optional String representing a Wei number to override the disputer price feed.
  * @return None or throws an Error.
  */
-async function run(
+async function run({
   logger,
```

Comment:
nit: I know this is super annoying work but can you update the comments? Doesn't have to be detailed but helps for code editors that read the Natspec comments